perseo-utils/src/test/java/com/telefonica/iot/perseo/utils/DateTimeUtilsTest.java

```diff
@@ -51,27 +55,35 @@ public void testGetNextSunset() {
     }
 
     /**
-     * Test get milis to next sunrise.
+     * Test get milis to next sunrise
+     * Calculates milliseconds to next sunrise considering current date
+     * Is successful if the resoult is less than 24 hrs (86400000 milliseconds)
      */
     public void testGetMilisToNextSunrise() {
         System.out.println("getMilisToNextSunrise");
+        long dayToMilis = 86400000;
```

Comment:
It is used twice. Maybe this can be moved to a class constant, so every method has it available.

Reply:
Fixed in [01b0229](https://github.com/telefonicaid/perseo-core/pull/146/commits/01b0229f807458f7ad9697a38233459dd3ddccd6)